ui/components/app/metamask-template-renderer/safe-component-list.js

```diff
@@ -121,7 +120,6 @@ export const safeComponentList = {
   TextField,
   Tooltip,
   TruncatedDefinitionList,
-  Typography,
```

Comment:
This could be considered a breaking change if snaps allows the Typography component or used by dapps for confirmations. Will need to check with @MetaMask/snaps-devs and @MetaMask/confirmations